lms/djangoapps/courseware/courses.py

```diff
@@ -108,6 +108,17 @@ def get_course_overview_with_access(user, action, course_key, check_if_enrolled=
     return course_overview
 
 
+def is_enrolled_in_course_or_staff(course_key, user):
+    """
+    Returns true if the user is enrolled in the specified course.
+
+    Note: users with staff access return True even if not enrolled.
+    """
+    if user.is_anonymous():
```

Comment:
Good point. This check doesn't seem to be necessary as both methods handle anonymous users. I'm not sure why I thought it was.